userspace/libsinsp/parsers.cpp

```diff
@@ -2289,9 +2289,7 @@ void sinsp_parser::parse_open_openat_creat_exit(sinsp_evt *evt)
 			ASSERT(parinfo->m_len == sizeof(uint32_t));
 			enter_evt_flags = *(uint32_t *)parinfo->m_val;
 
-			if(enter_evt_namelen != 0
-			   && strncmp(enter_evt_name, "(NULL)", 7) != 0
-			   && strncmp(enter_evt_name, "<NA>", 5) != 0)
+			if(enter_evt_namelen != 0 && strncmp(enter_evt_name, "<NA>", 5) != 0)
```

Comment:
removed the `(NULL)` case for clarity, now we will manage everything with `<NA>`